docker/Dockerfile-nvidia-10

```diff
@@ -0,0 +1,32 @@
+FROM nvidia/cuda:10.1-runtime-ubuntu18.04
```

Comment:
Why CUDA 10.1 and not 11.x?

Reply:
At one point I was having issues with CUDA 11 and the openmm version I was using, 11.1 works just fine and I've updated the container.